packages/web/src/components/table/components/TablePlayButton.tsx

```diff
@@ -24,15 +24,15 @@ export const TablePlayButton = ({
   paused,
   playing = false,
   isTrackPremium = false,
-  isLocked = false
+  isOwned = false
```

Comment:
tbh i'm kind of leaning towards not renaming this to stay consistent with existing patterns, esp since we're setting `isOwned={!isLocked}` in `TracksTable`. To me the logic goes: it's locked if i don't have access, and if i'm the owner i'll have access.